tests/chainsaw/loadbalancer-services/test-tls-custom-domain-service.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: test-tls-custom-domain-lb
+  annotations:
+    k8s.ngrok.com/url: tls://service-lb-tls-test-ngrok-operator.example.com:443
```

Comment:
This test fails in the e2e merge queue. I picked a domain i thought would be fairly custom and not used to avoid conflicts with future clients. However, the issue is that i run it locally so my account reserved the domain and now the CI one will fail. I can't just unreserve it as we'll then face the same issue in our local systems. 

Instead i'm going to just comment this test out for now and will investigate how we can ensure these domains are custom either by having chainsaw create a unique namespace for each run and use that in its templating, or by injecting a unique value from the PR number or codespace